spec/integration/outputs/templates_spec.rb

```diff
@@ -22,7 +22,7 @@
 
   before :each do
     # Delete all templates first.
-    require "elasticsearch"
+    require "opensearch"
```

Comment:
Why does this need to be required at all if no code references it?

Reply:
Ack.